.github/workflows/build.yml

```diff
@@ -46,7 +50,8 @@ jobs:
     # and co-authored by the user that made the last commit.
     # https://github.com/marketplace/actions/git-auto-commit
     - name: Commit the changes to the binary files
-      if: false  # node-gyp builds are not reproducible, hence each build would create a "new" commit -> disabling for now (comment out this line if needed)
+      # if: true  # node-gyp builds are not reproducible, hence each build would create a "new" commit -> disabling for now (comment out this line if needed)
+      if: github.event_name == 'release' && github.event.action == 'created'  # commit new binaries package on releases
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-contexts